sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_post_batch_rejected.py

```diff
@@ -3,12 +3,12 @@
 from sovtoken import TOKEN_LEDGER_ID
 
 
-def test_token_batch_handler_post_batch_rejected(token_batch_handler, utxo_cache):
+def test_token_batch_handler_post_batch_rejected(utxo_batch_handler, utxo_cache):
```

Comment:
A correct test name is `test_utxo_batch_handler_post_batch_rejected`

Reply:
fixed